make/photon/prepare/templates/core/env.jinja

```diff
@@ -18,6 +18,7 @@ POSTGRESQL_SSLMODE={{harbor_db_sslmode}}
 POSTGRESQL_MAX_IDLE_CONNS={{harbor_db_max_idle_conns}}
 POSTGRESQL_MAX_OPEN_CONNS={{harbor_db_max_open_conns}}
 REGISTRY_URL={{registry_url}}
+PORTAL_URL={{portal_url}}
```

Comment:
Why core need to know the url of portal?

Reply:
used for the health check on core